war/.eslintrc.js

```diff
@@ -62,21 +64,13 @@ module.exports = {
     setupWizardExtensions: "readonly",
     SharedArrayBuffer: "readonly",
     shortenName: "readonly",
+    Sortable: "readonly",
     toQueryString: "readonly",
     ts_refresh: "readonly",
     updateOptionalBlock: "readonly",
+    Utilities: "readonly",
+    UTILITIES_VERSION: "writeable",
     YAHOO: "readonly",
-
-    __dirname: false,
-
-    // Allow jest globals used in tests
-    jest: false,
-    expect: false,
-    it: false,
-    describe: false,
-    beforeEach: false,
-    afterEach: false,
-    beforeAll: false,
-    afterAll: false,
```

Comment:
Cleaning up after #6908 which removed `jest.config.js` but forgot to remove these exclusions.